tasks/gen/model.cr

```diff
@@ -0,0 +1,49 @@
+class Gen::Model < LuckyCli::Task
+  banner "Generate a model, query, and form"
+  getter io : IO = STDOUT
+
+  def call(@io : IO = STDOUT)
+    if valid?
+      template.render("./src/")
+      display_success_messages
+    else
+      io.puts @error.colorize(:red)
+    end
+  end
+
+  private def valid?
+    model_name_is_present && model_name_is_camelcase
+  end
+
+  private def model_name_is_present
+    @error = "Model name is required. Example: lucky gen.model User"
+    ARGV.first?
+  end
+
+  private def model_name_is_camelcase
+    @error = "Model name should be camel case. Example: lucky gen.model ContactInfo"
+    model_name.camelcase == model_name
```

Comment:
Could you use the model name in the error message, “lucky gen.model #{model_name.camelcase}”

Reply:
Great idea